### ISSUE TITLE
QUESTION ABOUT BOTS

### ISSUE THREAD

**Issue by nitrosoft1** [instabot-py/instabot.py#1843]
Hello, 

I hope you are doing well, i have two questions :
1- i used the instabot for 2 weeks, and now i saw that i cannot follow "it's blocked by instagram" what is the solution ? i can only like and it's not very good..
2- Can we use multiple bots for the same account ? and how can we do it ? thank you very much

**Comment by simonmanuel** [comment]
Hi there,

I may not be technically qualified, but I can answer based on my experience from the few months that I've been using the bot.

1. When something like that happens, it's better to cool off for a week or two. In the meantime, you could report the action on the app when Instagram asks you to _report a problem_ for said action.

2. Using multiple bots for the same account is not advisable. What purpose does using multiple bots serve?

To be on the safer side, always ramp up your follow/unfollow/like numbers. 100 the first week, then 200. I've used upto 800 a day without being blocked and I've even seen some push it 3000 a day. Incremental increases would make it look more 'human'.

Hope this helps.

**Comment by nitrosoft1** [comment]
Thank you very much,

You seem that you know lot of informations about instagram bot , i have many problems with mine ... can we exchange our email or any method to discuss easier ? 

I really need help lol thank you very much !

**Comment by simonmanuel** [comment]
Hey @nitrosoft1 I'm not well versed with how this thing works. Most of it is just trial and error. Any issue you run into is almost always fixable by just googling it. You can cut a middleman by yourself, yea!

**Comment by DarkSuniuM** [comment]
Closed all the issues till 2019-01-09, We updated the repo, Please create a new issue if you need! Thanks
